### DISCUSSION TITLE
Focus on core functionalities and usecases

### DISCUSSION THREAD

**Discussion by FoFighter20** [powerpaul17/nc_money#66] [Ideas]
Hi @powerpaul17 

 I appreciate your efforts and I am fully aware that you are the only maintainer and your focus might have shifted away from this project. Please do not understand my remarks as a demand. Also, I am neither an accountant or a developer/maintainer. I am not very experienced with the cultural aspects of github and so on and so forth. The only Programming Language I would say I know is python. So please understand my remarks as a humble proposal for improvement rather than demands!
 
That being said, the app is currently not fulfilling what **I** expected from a personal finance app. Double bookkeeping is relevant for businesses to keep track how "money is being exchanged". The balance sheet needs to net zero (i.e. has to be balanced) as all incoming transactions must have a outgoing transaction from a different account. Selling for example a car would reduce the amount of cars, which would pop up as a negative number on the balance sheet. This would be re-balanced by the influx of cash to net zero. 
In my opinion this doesn't really make sense for individuals, which is the reason why accounting seems more complicated than it actually is because it is very artificial. 

So, as an individual what would be my personal goal?
- I would like to asses my net assets and liabilities in the future to extrapolate a time at which a certain asset could be bought
- I would like to understand how saving money impacts my net worth over time.

Seeing the app in its current state I am not sure how double book keeping helps me in achieving these goals.
Instead these functions would be what I expect from such an app:
1. Planning: Enabling the user to set up (reoccurring) in and outgoing transactions.
2. Monitoring: Enabling the user to parse data from their bank account.
3. Transactions: Enabling the user to handle transactions and tagging the transactions with metadata/categories.
4. Filtering: Enabling the user to add filters that automatically tag transactions from parsed bank accounts based on rules.
5. Accounts: Enable the user to set up separate accounts such as a savings account, depot or a loan where interest rates can be adjusted separately.

What do you think about this?

Kind regards
 Raoul

**Comment by powerpaul17** [comment]
Hello and thanks for your post.

As you already noticed I am the only developer/maintainer of this app and unfortunately my time is very limited. So there are rather long gaps between releases.

Short answer: all of your suggestions are planned and some of them already exist as issues here (some I have on my personal list)

Long answer:

1. is planned
2. is already possible through CSV-Import. Directly importing from banks is not possible because unfortunately there is no standardized API for doing so worldwide.
3. is planned
4. is planned
5. you can already create as many accounts as you want. Interest rates are planned and also tracked by #32 

I am fully aware that this app lacks a lot of desireable features but time is really a constraint here, since I also have to earn money and take care of my family :slightly_smiling_face:

**Comment by powerpaul17** [comment]
I want to add that parts of point 3 are already possible: categories in your case would simply be income/expense accounts.
